commands/cli/helptext.go

```diff
@@ -169,7 +211,20 @@ func LongHelp(rootName string, root *cmds.Command, path []string, out io.Writer)
 }
 
 // ShortHelp returns a formatted CLI helptext string, generated for the given command
-func ShortHelp(rootName string, root *cmds.Command, path []string, out io.Writer) error {
+func ShortHelp(rootName string, root *cmds.Command, req cmds.Request, path []string, out io.Writer) error {
+	terminalColors := false
+	if req != nil {
```

Comment:
How to read off the config's flag from here? I thought it has to be:
```
  ctx := invoc.req.Context()
  cfg, _ := ctx.GetConfig()
  // we get cfg.Preferences.TerminalColors
```
but `invoc.req` is not even initialized by default.